cdap-ui/app/features/admin/templates/namespace/datasets.html

```diff
@@ -3,43 +3,36 @@
     <div class="col-xs-6">
       <h2> Datasets </h2>
     </div>
-    <div class="col-xs-6" ng-if="datasets.length > 0 || streams.length > 0">
+    <div class="col-xs-6" ng-if="dataList.length > 0">
       <button class="btn btn-default pull-right" ui-sref="admin.namespace.detail.data.streamcreate({nsadmin: $stateParams.nsadmin})"><span class="fa fa-plus"></span> Add Stream</button>
     </div>
   </div>
-  <div ng-if="datasets.length > 0 || streams.length > 0" datasets-list>
+  <div ng-if="dataList.length > 0" datasets-list>
     <div class="table-responsive">
       <table class="table table-curved" cask-sortable>
         <thead>
-          <tr ng-class="{'sort-enabled': datasets.length>0}">
+          <tr ng-class="{'sort-enabled': dataList.length>0}">
             <th class="col-xs-5" data-predicate="name">Name</th>
             <th class="col-xs-7" data-predicate="type">Type</th>
 
           </tr>
         </thead>
         <tbody>
-          <tr ng-repeat="dataset in datasets | orderBy:sortable.predicate:sortable.reverse">
-            <td class="col-xs-5">
-              <strong class="icon-datasets"></strong>
-              <a ui-sref="admin.namespace.detail.data.datasetmetadata({datasetId: dataset.name})">
-
-                <strong ng-bind="dataset.name"></strong>
+          <tr ng-repeat="data in dataList | orderBy:sortable.predicate:sortable.reverse">
```

Comment:
If clicking the column A, then it will sort by that column. Clicking that column A again will reverse the elements, only if you have `sortable.reverse`. Without that, it won't sort in reverse order.